controllers/storageclusterpeer/storageclusterpeer_controller.go

```diff
@@ -18,46 +18,176 @@ package storageclusterpeer
 
 import (
 	"context"
+	"fmt"
+	"time"
 
+	ocsv1 "github.com/red-hat-storage/ocs-operator/api/v4/v1"
+	providerClient "github.com/red-hat-storage/ocs-operator/services/provider/api/v4/client"
+
+	"github.com/go-logr/logr"
+	"google.golang.org/grpc/status"
+	"k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/runtime"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
 
-	v1 "github.com/red-hat-storage/ocs-operator/api/v4/v1"
+const (
+	storageClusterPeerFinalizer = "storageclusterpeer.ocs.openshift.io"
 )
 
 // StorageClusterPeerReconciler reconciles a StorageClusterPeer object
 // nolint:revive
```

Comment:
nolint marker can be removed now?

Reply:
This is becasue the struct name starts with same as package name

Reply:
are you sure? the names are different, isn't it?

Reply:
Yes, the struct name is StorageClusterPeerReconciler and package name is storageclusterpeer

Reply:
Ooh, you mean `starts with.... `, missed it, I don't get what linter is trying to enforce though 😅. Resolve by adding a comment in code as well.